packages/url_launcher/url_launcher_platform_interface/pubspec.yaml

```diff
@@ -0,0 +1,17 @@
+name: url_launcher_platform_interface
+description: A common platform interface for the url_launcher plugin.
+author: Flutter Team <flutter-dev@googlegroups.com>
+homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher/url_launcher_platform_interface
+version: 1.0.0
```

Comment:
I'd keep the comment here as well, I feel like ongoing maintainers are less likely to page in the README before publishing, but they will visit this line for sure if they want to bump the major version.

Reply:
Done.